<node name="gps"
	    pkg="autogator" type="gps.py">
	  <remap from="gps_location" to="~output" />
	</node>

	<node name="em_supervision"
        pkg="autogator" type="em_supervision.py">
    <rosparam>
      msg_name: std_msgs/String
    </rosparam>
  </node>

  <test name="publishtest"
        test-name="publishtest"
        pkg="autogator" type="publishtest">
    <rosparam>
      topics:
        - name: gps/output
          timeout: 10
          negative: False
        - name: em_supervision/output
          timeout: 3
          negative: True  # means it is not published until the timeout
    </rosparam>
  </test>